sru/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.3.5'
+__version__ = '2.4.0'
```

Comment:
Minor version sounds like the appropriate version number to change to me :) (assuming this change is backwards-compatible? which I think, it is?)